orangecontrib/text/widgets/owsentimentanalysis.py

```diff
@@ -0,0 +1,75 @@
+from AnyQt.QtWidgets import QApplication
+
+from Orange.widgets import gui, settings
+from Orange.widgets.utils.signals import Input, Output
+from Orange.widgets.widget import OWWidget
+from orangecontrib.text import Corpus
+from orangecontrib.text.sentiment import Vader_Sentiment, Liu_Hu_Sentiment
+
+METHODS = [Liu_Hu_Sentiment(),
+           Vader_Sentiment()]
+
+class OWSentimentAnalysis(OWWidget):
+    name = "Sentiment Analysis"
+    description = "Predict sentiment from text."
+    icon = "icons/SentimentAnalysis.svg"
+    priority = 30001
+
+    class Inputs:
+        corpus = Input("Corpus", Corpus)
+
+    class Outputs:
+        corpus = Output("Corpus", Corpus)
+
+    method_idx = settings.Setting(1)
+    autocommit = settings.Setting(True)
+    want_main_area = False
+
+    def __init__(self):
+        super().__init__()
+        self.corpus = None
+
+        gui.radioButtons(self.controlArea, self, "method_idx", box="Method",
+                         btnLabels=[m.name for m in METHODS],
+                         callback=self._method_changed)
+
+        ac = gui.auto_commit(self.controlArea, self, 'autocommit', 'Commit',
+                             'Autocommit is on')
+        ac.layout().insertWidget(0, self.report_button)
+        ac.layout().insertSpacing(1, 8)
+
+        gui.rubber(self.controlArea)
```

Comment:
Having rubber at the bottom feels weird.
But since the content of this widget is fixed and there is no need for resizing, it would be best to making it fixed-sized anyway (like Data Info, all widgets from Model, etc).